### ISSUE TITLE
Add zh_cn launguage

### ISSUE THREAD

**Issue by hyplant** [mega12345mega/NBT-Editor#66]
希望添加简体中文语言
It is supposed to add Simplified Chinese language, here is the language file I made for reference.

```
{
	"nbteditor.lang.zh_cn.version": "0.1.2",
	"nbteditor": "NBT 编辑器",
	"nbteditor.outdated.title": "NBT Editor 版本过旧",
	"nbteditor.outdated.desc": "有一个新版本已发布。",
	
	
	"nbteditor.ok": "确定",
	"nbteditor.cancel": "取消",
	
	"nbteditor.no_hand.no_item.to_edit": "没有手持物品可供编辑！",
	"nbteditor.no_hand.no_item.to_view": "没有手持物品可供查看！",
	"nbteditor.no_hand.no_item.to_export": "没有手持物品可供导出！",
	"nbteditor.no_hand.no_item.container": "没有手持容器可供打开！",
	"nbteditor.no_hand.no_item.skull": "没有手持玩家模型可供编辑！",
	"nbteditor.no_hand.no_item.block": "没有手持方块可供编辑！",
	"nbteditor.no_hand.no_item.book": "没有手持成书可供编辑！",
	"nbteditor.no_hand.no_item.signboard": "没有手持告示牌可供编辑！",
	"nbteditor.no_hand.all_item": "没有空槽位可供添加新物品！",
	
	"nbteditor.editor.save": "保存物品",
	"nbteditor.editor.saving": "正在保存…",
	"nbteditor.editor.name": "名称",
	
	"nbteditor.editor.unsaved.title": "物品未保存",
	"nbteditor.editor.unsaved.desc": "尚未保存物品，是否要保存？",
	"nbteditor.editor.unsaved.yes": "保存",
	"nbteditor.editor.unsaved.no": "放弃更改",
	
	"nbteditor.nbt.empty_key.title": "不支持的物品",
	"nbteditor.nbt.empty_key.desc": "当前编辑器不支持有空字符串（\"\"）作为键名的物品。是否要移除该键值对？",
	"nbteditor.nbt.empty_key.yes": "修复并编辑",
	"nbteditor.nbt.empty_key.no": "取消",
	
	"nbteditor.nbt.count": "数量",
	"nbteditor.nbt.identifier": "命名空间ID",
	"nbteditor.nbt.path": "路径",
	"nbteditor.nbt.value": "值",
	"nbteditor.nbt.value_expand": "展开",
	"nbteditor.nbt.add": "§a+",
	"nbteditor.nbt.remove": "§cx",
	"nbteditor.nbt.copy": "复制",
	"nbteditor.nbt.cut": "剪切",
	"nbteditor.nbt.paste": "粘贴",
	"nbteditor.nbt.rename": "重命名",
	
	"nbteditor.nbt.overwrite.title": "该名称已存在",
	"nbteditor.nbt.overwrite.desc": "是否要覆盖它？",
	"nbteditor.nbt.overwrite.yes": "覆盖",
	"nbteditor.nbt.overwrite.no": "取消",
	
	"nbteditor.nbt.saving_air.title": "数据将丢失",
	"nbteditor.nbt.saving_air.desc": "正在保存空气，这将丢失数据。是否继续保存？",
	"nbteditor.nbt.saving_air.yes": "放弃数据并保存",
	"nbteditor.nbt.saving_air.no": "取消",
	
	"nbteditor.nbt.export.copied": "§a命令已复制到剪贴板！",
	"nbteditor.nbt.export.file.success": "物品已作为 %s 保存",
	"nbteditor.nbt.export.file.error": "无法保存物品：%s",
	
	"nbteditor.file_options.show": "§a[打开文件夹]",
	"nbteditor.file_options.delete": "§c[删除]",
	"nbteditor.file_options.delete.title": "%s",
	"nbteditor.file_options.delete.desc": "确认要删除吗？该操作无法撤销！",
	"nbteditor.file_options.delete.success": "§a已删除 %s",
	"nbteditor.file_options.delete.error": "§c删除 %s 时出现错误",
	"nbteditor.file_options.delete.missing": "§c%s§c 不存在，可能已被删除。",
	
	"nbteditor.img2lore": "导入图片",
	
	"itemGroup.nbteditor.client_chest": "客户端物品栏",
	"itemGroup.nbteditor.creative": "创造模式物品栏",
	"nbteditor.client_chest": "Nbt Editor >> 客户端物品栏",
	"nbteditor.client_chest.save_error": "§c保存客户端物品栏时出现错误！",
	"nbteditor.client_chest.not_ready": "§c客户端物品栏加载中，可能因为数据太多。",
	"nbteditor.client_chest.corrupt_warning": "§c某些页面可能被损坏，已被分开备份。",
	"nbteditor.client_chest.slots.lock": "锁定槽位",
	"nbteditor.client_chest.slots.unlock": "解锁槽位",
	"nbteditor.client_chest.reload_page": "重新加载页面",
	"nbteditor.client_chest.clear_page": "清空页面",
	
	"nbteditor.client_chest.clear_page.title": "确认清空页面？",
	"nbteditor.client_chest.clear_page.desc": "这将删除本页保存的54个物品！",
	"nbteditor.client_chest.clear_page.yes": "删除物品",
	"nbteditor.client_chest.clear_page.no": "取消",
	
	"nbteditor.book.convert": "转为可编辑状态",
	"nbteditor.book.convert.formatting_saved": "该成书包含格式化代码，在可编辑书中不可用。将备份并转换该成书。",
	
	"nbteditor.container": "容器",
	"nbteditor.container.title": "Nbt Editor >> ",
	
	"nbteditor.item_factory": "高级编辑器",
	
	"nbteditor.sign.added": "§a已添加签名",
	"nbteditor.sign.removed": "§a已移除签名",
	"nbteditor.sign.edited": "§a已修改签名",
	"nbteditor.sign.default": "§3默认签名！请使用命令 /sign edit <新签名>",
	"nbteditor.sign.already_added": "§c该物品已有您的签名！",
	"nbteditor.sign.not_added": "§c该物品没有你的签名！",
	"nbteditor.sign.new.missing_arg": "§c你需要创建新签名！请使用命令 /sign edit <新签名>",
	"nbteditor.sign.save_error": "§c保存签名时出现错误！",
	"nbteditor.sign.load_error": "§c加载签名时出错！重启可能解决",
	
	"nbteditor.lore.edited": "§a附加信息已编辑！",
	"nbteditor.lore.none": "§c没有附加信息！",
	
	"nbteditor.img_to_lore.width": "宽度",
	"nbteditor.img_to_lore.height": "高度",
	
	"nbteditor.get.item": "§a获取：",
	"nbteditor.get.lost_item": "§c你可能失去物品 - ",
	"nbteditor.get.lost_item.hover": "点击来执行",
	"nbteditor.get.lost_item.none": "§c没有保存任何丢失的物品！",
	"nbteditor.get.lost_item.history_hint": "§a使用 [run_command]{/get lostitem history}([show_text]{Click to execute}(§6/get lostitem history)) 来查看最近选中的物品。",
	"nbteditor.get.lost_item.history": "选中物品历史",
	"nbteditor.get.preset_item.missing": "§c预设物品加载失败！",
	
	"nbteditor.hide_flags": "隐藏标记",
	"nbteditor.hide_flags.hide_all": "§c全部隐藏",
	"nbteditor.hide_flags.show_all": "§a全部显示",
	"nbteditor.hide_flags.enchantments": "附魔",
	"nbteditor.hide_flags.attribute_modifiers": "属性修饰符",
	"nbteditor.hide_flags.unbreakable": "无法破坏",
	"nbteditor.hide_flags.can_destroy": "可挖掘",
	"nbteditor.hide_flags.can_place_on": "可放置",
	"nbteditor.hide_flags.misc": "杂项",
	"nbteditor.hide_flags.dyed_color": "盔甲染色",
	
	"nbteditor.keybind.enchant": "按住 §6§lCTRL§r§6 点击来附魔",
	"nbteditor.keybind.edit": "按 §6§l空格§r§6 来编辑",
	"nbteditor.keybind.container": "按 §6§lCTRL + 空格§r§6 来打开容器",
	"nbteditor.keybind.item_factory": "按 §6§lShift + 空格§r§6 打开高级编辑器",
	
	"nbteditor.unbind_skull.no_textures": "该玩家模型未加载任何贴图",
	"nbteditor.unbind_skull.unbound": "§a玩家模型未绑定！",
	
	"nbteditor.maxed": "§a附魔已达最高等级！",
	
	"nbteditor.block_states": "方块状态",
	"nbteditor.block_states.none": "该方块没有方块状态！",
	
	"nbteditor.attributes": "属性修饰符",
	"nbteditor.attributes.attribute": "实体属性",
	"nbteditor.attributes.operation": "操作方式",
	"nbteditor.attributes.amount": "数值",
	"nbteditor.attributes.amount.max": "最大",
	"nbteditor.attributes.amount.min": "最小",
	"nbteditor.attributes.amount.infinity": "∞",
	"nbteditor.attributes.amount.negative_infinity": "-∞",
	"nbteditor.attributes.amount.autofill_keybinds": "按住 SHIFT 无限\n按住 CTRL 最小",
	"nbteditor.attributes.slot": "可用槽位",
	"nbteditor.attributes.new_uuids.success": "§a已随机化属性 UUID",
	"nbteditor.attributes.new_uuids.no_attributes": "§c该物品没有属性修饰符！",
	
	"nbteditor.enchantments": "物品附魔",
	"nbteditor.enchantments.enchantment": "附魔",
	"nbteditor.enchantments.level": "等级",
	"nbteditor.enchantments.tip": "你可以在支持的容器中按住 CTRL 使用附魔书点击物品来附魔",
	
	"nbteditor.unstackable.msg": "§a物品现在无法堆叠（放上鼠标了解详情）",
	"nbteditor.unstackable.hover_msg": "已添加随机 UUID。\n保存该物品时，\n请为所有个人物品\n重新运行该命令。",
	
	"nbteditor.named": "§a品名称已更改为：",
	
	"nbteditor.display": "显示",
	
	"nbteditor.book": "书",
	"nbteditor.book.open": "编辑书",
	"nbteditor.book.title": "标题",
	"nbteditor.book.author": "作者",
	"nbteditor.book.page": "页码：%d/%d",
	"nbteditor.book.back": "上一页",
	"nbteditor.book.forward": "下一页",
	"nbteditor.book.add": "§a添加",
	"nbteditor.book.remove": "§c删除",
	"nbteditor.book.preview": "预览",
	"nbteditor.book.preview.icon": "成书预览",
	"nbteditor.book.preview.click.title": "§l超链接",
	"nbteditor.book.preview.click.action": "动作类型：%s",
	"nbteditor.book.preview.click.value": "动作值：%s",
	
	"nbteditor.signboard": "告示牌（方块）",
	"nbteditor.signboard.side.front": "正面",
	"nbteditor.signboard.side.back": "背面",
	"nbteditor.signboard.wax.enabled": "已涂蜡",
	"nbteditor.signboard.wax.disabled": "未涂蜡",
	"nbteditor.signboard.glowing.enabled": "发光",
	"nbteditor.signboard.glowing.disabled": "不发光",
	"nbteditor.signboard.import.success": "§a告示牌更改为：%s",
	"nbteditor.signboard.import.fail": "§c告示牌已经是 %s",
	
	"nbteditor.multi_line_text.find": "查找",
	"nbteditor.multi_line_text.replace": "替换",
	"nbteditor.multi_line_text.replace_all": "全部替换",
	"nbteditor.multi_line_text.x": "X",
	"nbteditor.multi_line_text.regex": "正则表达式",
	"nbteditor.multi_line_text.regex.on": "使用正则表达式",
	"nbteditor.multi_line_text.regex.off": "禁用正则表达式",
	"nbteditor.formatted_text.custom_color": "自定义颜色",
	"nbteditor.formatted_text.events": "悬浮和点击行为",
	"nbteditor.formatted_text.click_event_value": "点击行为",
	"nbteditor.formatted_text.hover_event_value": "悬浮行为",
	"nbteditor.formatted_text.tip": "按 Ctrl+Shift+C 自定义颜色，按 Ctrl+Shift+E 自定义事件",
	
	"nbteditor.config": "NBT Editor 设置",
	"nbteditor.config.category.mc": "Minecraft 调整",
	"nbteditor.config.category.guis": "图形用户界面调整",
	"nbteditor.config.category.functional": "功能性调整",
	"nbteditor.config.shortcuts": "快捷方式",
	"nbteditor.config.shortcuts.desc": "为命令创建快捷方式，例如：\n使用 /lore 代替 /itemfactory display lore",
	"nbteditor.config.shortcuts.example": "例子：为 \"/get hdb\" 创建快捷方式 \"/hdb\"",
	"nbteditor.config.enchant_level_max": "显示最高附魔等级",
	"nbteditor.config.enchant_level_max.desc": "在附魔等级之后显示该项目的最大值",
	"nbteditor.config.enchant_level_max.never": "禁用",
	"nbteditor.config.enchant_level_max.not_exact": "仅在小于最大时",
	"nbteditor.config.enchant_level_max.not_max": "仅在非最大时",
	"nbteditor.config.enchant_level_max.always": "始终显示",
	"nbteditor.config.enchant_number_type": "附魔显示方式",
	"nbteditor.config.enchant_number_type.desc": "使用指定的方式\n显示附魔等级",
	"nbteditor.config.enchant_number_type.desc2": "阿拉伯数字：1-10\n罗马数字：I-X",
	"nbteditor.config.enchant_number_type.arabic": "阿拉伯数字",
	"nbteditor.config.enchant_number_type.roman": "罗马数字",
	"nbteditor.config.key_text_size": "按键大小",
	"nbteditor.config.key_text_size.desc": "调整编辑器列表中按键的大小",
	"nbteditor.config.keybinds": "显示按键绑定",
	"nbteditor.config.keybinds.desc": "是否在物品提示中显示按键绑定",
	"nbteditor.config.keybinds.hidden": "隐藏",
	"nbteditor.config.keybinds.shown": "显示",
	"nbteditor.config.chat_limit": "聊天栏限制",
	"nbteditor.config.chat_limit.desc": "扩展聊天栏中命令长度的限制",
	"nbteditor.config.chat_limit.normal": "默认",
	"nbteditor.config.chat_limit.extended": "扩展",
	"nbteditor.config.single_quotes": "使用单引号",
	"nbteditor.config.single_quotes.desc": "在键和字符串外使用单引号，\n可以缩短 NBT，\n但可能导致字符串出现问题",
	"nbteditor.config.single_quotes.example": "启用时：\"{§e'§rkey with quote -> §e\\§r\" <-§e'§r:0}\"\n禁用时：'{§e\"§rkey with quote -> §e\\\\§r\" <-§e\"§r:0}'",
	"nbteditor.config.single_quotes.allowed": "启用",
	"nbteditor.config.single_quotes.not_allowed": "禁用",
	"nbteditor.config.mac_scroll_patch": "MacOS 边缘补丁",
	"nbteditor.config.mac_scroll_patch.on_mac": "§cMacOS 边缘补丁",
	"nbteditor.config.mac_scroll_patch.desc": "一些 Mac 用户反馈滚动时边缘出现问题，\n如果您发现了问题，请启用该选项，\n这个补丁并不完美，可能导致其他问题",
	"nbteditor.config.mac_scroll_patch.enabled": "启用",
	"nbteditor.config.mac_scroll_patch.disabled": "禁用",
	"nbteditor.config.scroll_speed": "滚动速度",
	"nbteditor.config.scroll_speed.desc": "提高或降低 NBT 编辑菜单的滚动速度",
	"nbteditor.config.air_editable": "编辑空气",
	"nbteditor.config.air_editable.desc": "允许编辑物品栏中的空气，\n§e实验性功能！\n§c使用前请先备份重要的物品",
	"nbteditor.config.air_editable.yes": "启用",
	"nbteditor.config.air_editable.no": "禁用",
	"nbteditor.config.json_text": "原始 json 文本",
	"nbteditor.config.json_text.desc": "使用原始 json 文本代替格式化代码\n通过命令 /get help textformat 了解详情",
	"nbteditor.config.json_text.yes": "启用",
	"nbteditor.config.json_text.no": "禁用",
	"nbteditor.config.check_updates": "检查更新",
	"nbteditor.config.check_updates.desc": "是否在启动时检查并显示更新通知",
	"nbteditor.config.check_updates.minor": "仅限大更新",
	"nbteditor.config.check_updates.minor.desc": "仅显示大更新，会包含若干新功能",
	"nbteditor.config.check_updates.patch": "显示小更新",
	"nbteditor.config.check_updates.patch.desc": "也显示小更新，可能只修复一些漏洞或新增少数功能",
	"nbteditor.config.check_updates.none": "不显示更新",
	"nbteditor.config.check_updates.none.desc": "不显示任何更新",
	"nbteditor.config.client_chest_size": "客户端物品栏容量",
	"nbteditor.config.client_chest_size.desc": "把客户端物品的容量从100页提升到无限\n在大量页面中滚动时可能造成卡顿\n可能不会增加内存占用",
	"nbteditor.config.client_chest_size.small": "小容量",
	"nbteditor.config.client_chest_size.large": "大容量",
	"nbteditor.config.screenshot_options": "截图额外选项",
	"nbteditor.config.screenshot_options.desc": "将 %s §r和 %s 选项\n§r添加到截图消息中",
	"nbteditor.config.screenshot_options.enabled": "启用",
	"nbteditor.config.screenshot_options.disabled": "禁用",
	"nbteditor.config.tooltip_overflow_fix": "工具提示浮窗修复",
	"nbteditor.config.tooltip_overflow_fix.desc": "防止工具提示浮窗超出屏幕\n如果有需要，这将把工具提示浮窗下移",
	"nbteditor.config.tooltip_overflow_fix.enabled": "启用",
	"nbteditor.config.tooltip_overflow_fix.disabled": "禁用",
	"nbteditor.config.no_armor_restriction": "移除盔甲栏限制",
	"nbteditor.config.no_armor_restriction.desc": "允许你将非盔甲物品放入盔甲栏",
	"nbteditor.config.no_armor_restriction.enabled": "启用",
	"nbteditor.config.no_armor_restriction.disabled": "禁用",
	"nbteditor.config.hide_format_buttons": "隐藏格式按钮",
	"nbteditor.config.hide_format_buttons.desc": "隐藏文本格式编辑器中\n可通过快捷键使用的格式按钮\n\n这可以使先前没有空间显示的颜色列表正常展开",
	"nbteditor.config.hide_format_buttons.enabled": "隐藏",
	"nbteditor.config.hide_format_buttons.disabled": "显示",
	"nbteditor.config.special_numbers": "特殊数字",
	"nbteditor.config.special_numbers.desc": "允许在编辑器中输入特殊数字，例如：\n§6NaN§cd§r、§6Infinity§cf§r、§6-Infinity§cd§r\n这不被命令 /data 所接受（MC-200070）",
	"nbteditor.config.special_numbers.enabled": "启用",
	"nbteditor.config.special_numbers.disabled": "禁用",
	
	
	"nbteditor.configurable.list.move": "移动",
	"nbteditor.configurable.list.duplicate": "复制",
	"nbteditor.configurable.list.duplicate.any_amount": "按住 SHIFT 使用任意值",
	"nbteditor.configurable.list.remove": "移除",
	
	"nbteditor.effect_list_arg_type.expected.space": "展开空格",
	"nbteditor.effect_list_arg_type.expected.colon": "展开冒号（:）",
	"nbteditor.effect_list_arg_type.invalid.arg": "参数错误",
	"nbteditor.effect_list_arg_type.invalid.value": "值错误",
	
	"nbteditor.fancy_text_arg_type.invalid.event_type": "错误的鼠标悬浮或点击事件",
	"nbteditor.fancy_text_arg_type.invalid.entity": "该项目不存在",
	"nbteditor.fancy_text_arg_type.stringify_unsupported": "§c该文本包含无法被编辑器复现的格式或事件",
	
	"nbteditor.chat_length_warning_1": "警告：对于服务端命令，",
	"nbteditor.chat_length_warning_2": "长度限制为 256 个字符",
	
	
	
	"nbteditor.help_1": "§7--------------------",
	"nbteditor.help_2": "§6命令：",
	"nbteditor.help_3": "§6/nbt：§7编辑手持的物品",
	"nbteditor.help_4": "§6/chest：§7打开客户端物品栏",
	"nbteditor.help_5": "§6/open：§7打开手持的容器，支持在潜影盒中放入潜影盒",
	"nbteditor.help_6": "§6/get <item | potion | soup | skull | hdb | colorcodes | lostitem | help | credits> [<args>]：§7获取物品或物品信息",
	"nbteditor.help_7": "§6/sign [<add | remove | edit <new signature>>]：§7为物品签名",
	"nbteditor.help_8": "§6/itemfactory <factory> [<args>]：§7打开高级编辑器，可修改物品附魔、属性等",
	"nbteditor.help_9": "",
	"nbteditor.help_10": "§7获取具体功能帮助：§b/get help <nbteditor | clientchest | itemfactories>",
	"nbteditor.help_11": "§7发送反馈：",
	"nbteditor.help_12": "[链接] https://github.com/mega12345mega/NBT-Editor/issues",
	"nbteditor.help_13": "§7--------------------",
	
	"nbteditor.help.nbt_1": "§7--------------------",
	"nbteditor.help.nbt_2": "§6命令：/nbteditor、/nbt",
	"nbteditor.help.nbt_3": "§7编辑物品 NBT，包括物品自身及其数量",
	"nbteditor.help.nbt_4": "§7在名称输入框中，可以像铁砧一样重命名物品",
	"nbteditor.help.nbt_5": "§7与其所在列表不匹配的值（例如将数字放入字符串列表）将使用黄色高亮显示，并且不会被保存",
	"nbteditor.help.nbt_6": "§7更改项目类型时，可使用与 /data 相同的语法",
	"nbteditor.help.nbt_7": "§7例如：1b 表示字节型，2s表示短整型，3 表示整型，4L 表示长整型，5.6f 表示单精浮点数，7.8d 表示双精浮点数，\n[B;9,10]表示字节型数组，\"text\"表示字符串，[\"11.12\",\"text_text\"] 表示列表，{key:\"value\",number:13} 表示复合标签，[I;14,15] 表示整型数组，[L;16,17] 表示长整型数组",
	"nbteditor.help.nbt_8": "§7--------------------",
	
	"nbteditor.help.client_chest_1": "§7--------------------",
	"nbteditor.help.client_chest_2": "§6命令：/chest、/storage",
	"nbteditor.help.client_chest_3": "§7客户端物品栏在任何服务器和单人游戏中可用",
	"nbteditor.help.client_chest_4": "§7客户端物品栏会在修改时自动保存，保存时其标题将显示星号（*）",
	"nbteditor.help.client_chest_5": "§7你可以保存 100 页甚至更多物品",
	"nbteditor.help.client_chest_6": "§7点击侧边按钮前往下一个有内容的页面，所有页面为空时将前往首页/尾页",
	"nbteditor.help.client_chest_7": "§7锁定槽为后物品不能取出，非创造模式下始终开启，以防物品丢失",
	"nbteditor.help.client_chest_8": "§7重新加载将从文件中重新读取页面",
	"nbteditor.help.client_chest_9": "§7清空页面将删除该页面的所有物品，不可撤销。该选项有一个确认对话框以防误操作",
	"nbteditor.help.client_chest_10": "§7--------------------",
	
	"nbteditor.help.item_factories_1": "§7--------------------",
	"nbteditor.help.item_factories_2": "§6命令：/itemfactory <factory>",
	"nbteditor.help.item_factories_3": "§6<factory>可选值：",
	"nbteditor.help.item_factories_4": "§6attributes: §7编辑属性修饰符",
	"nbteditor.help.item_factories_5": "§6blockstates: §7编辑方块状态",
	"nbteditor.help.item_factories_6": "§6enchantments: §7编辑附魔",
	"nbteditor.help.item_factories_7": "§6hideflags: §7编辑隐藏标记",
	"nbteditor.help.item_factories_8": "§6lore <add | remove | set | clear | list> [<args>]: §7编辑附加信息",
	"nbteditor.help.item_factories_9": "§6max [cursed] [all] [<level>]: §7将附魔设为最高等级",
	"nbteditor.help.item_factories_10": "§6sign <add | remove | edit <new signature>>: §7创建新签名",
	"nbteditor.help.item_factories_11": "§6unbindskull: §7移除玩家模型绑定的玩家",
	"nbteditor.help.item_factories_12": "§7--------------------",
	
	"nbteditor.help.text_format_1": "§7--------------------",
	"nbteditor.help.text_format_2": "§6帮助：文本格式",
	"nbteditor.help.text_format_3": "§6使用原始 json 文本代替格式化代码",
	"nbteditor.help.text_format_4": "§6可以通过命令 /nbteditor config 更改设置",
	"nbteditor.help.text_format_5": "[格式] §6格式化代码：\\& for [run_command]{/get preset colorcodes}([show_text]{/get preset colorcodes}(§9§ncolor codes))",
	"nbteditor.help.text_format_6": "§6使用颜色代码时，为保证一致性，将清除格式",
	"nbteditor.help.text_format_7": "§6添加事件：[<event_name>]{<event_value>}(<affected text>)",
	"nbteditor.help.text_format_8": "§6例如：[run_command]{/say clicked!}(&6Click me!)",
	"nbteditor.help.text_format_9": "§6事件 [show_text] 需要格式化的文本作为值",
	"nbteditor.help.text_format_10": "§6事件 [show_item] 需要槽位编号作为值，默认为手持物品",
	"nbteditor.help.text_format_11": "§6事件 [show_entity] 需要实体 UUID 作为值，默认为指向的目标或你自己",
	"nbteditor.help.text_format_12": "§7--------------------",
	
	"nbteditor.credits_1": "§6NBT Editor 由 mega12345mega 创作",
	"nbteditor.credits_2": "§6希望你享受该模组！别用它惹恼§o太§r§6多 §c[管理员] §6:D",
	"nbteditor.credits_3": "",
	"nbteditor.credits_4": "§c§l玩家模型仓库§6由 UnlimitedNinjas 提供",
	"nbteditor.credits_5": "[链接] https://www.spigotmc.org/resources/free-headdb-head-menu.84967/",
	"nbteditor.credits_6": "§7由 mega12345mega 移植到 Fabric 客户端",
	
	
	
	"nbteditor.hdb.search": "搜索",
	"nbteditor.hdb.section": "区域",
	"nbteditor.hdb.head_not_found": "§c未找到玩家模型",
	"nbteditor.hdb.updating_database": "§6正在更新玩家模型仓库…",
	"nbteditor.hdb.updated_database": "§a玩家模型仓库已更新！",
	"nbteditor.hdb.unloaded_database": "§c正在加载玩家模型仓库…",
	"nbteditor.hdb.head_count": "%d 个玩家模型",
	
	"nbteditor.hdb.category.alphabet": "字母",
	"nbteditor.hdb.category.animals": "动物",
	"nbteditor.hdb.category.blocks": "方块",
	"nbteditor.hdb.category.decoration": "装饰",
	"nbteditor.hdb.category.food_drinks": "食物",
	"nbteditor.hdb.category.humans": "玩家",
	"nbteditor.hdb.category.humanoid": "人型生物",
	"nbteditor.hdb.category.miscellaneous": "杂项",
	"nbteditor.hdb.category.monsters": "怪物",
	"nbteditor.hdb.category.plants": "植物"
}

```

**Comment by mega12345mega** [comment]
This looks great! Thanks for translating. I'll include it in the next update.
I'm not sure the best way to keep new language entries up to date, but they should appear on the `dev` branch before being released.

**Comment by mega12345mega** [comment]
I've finally released this in 1.12.2! Thanks for the translation.

**Comment by mega12345mega** [comment]
I've just created a Discord server, which includes a channel for helping keep translations up to date. If you're still interested in translating, I'd appreciate you joining: [https://discord.gg/PzeYTbEZjn](https://discord.gg/PzeYTbEZjn)
Thanks!
